catalog/hierarchy/env-bu/hierarchy.yaml

```diff
@@ -12,12 +12,12 @@ spec:
     - prod:
         $subtree: environment
   parentRef:
-    external: "123456789012" # kpt-set: ${org-id}
+    # This should match your organization ID
+    external: "123456789012"
```

Comment:
I'd have same question. kpt-set seemed okay, just the default value in nested setters.yaml seemed to cause a problem...

That said - I guess it's almost but not quite idempotent that way. The nested resource generator would run before the LZ top level setters, so you'd need to render twice before folders would have correct org.

To avoid that would it be possible for `gcr.io/yakima-eap/generate-folders:latest` to paint `# kpt-set: ${org-id}` on the resulting `Folder.spec.organizationRef.external` properties?